--- b/xcbgen/xtypes.py.orig	2013-12-13 07:43:19.553165642 +0800
+++ b/xcbgen/xtypes.py	2013-12-13 07:43:32.436406957 +0800
@@ -76,7 +76,7 @@
         complex_type.fields.append(new_field)
 
     def make_fd_of(self, module, complex_type, fd_name):
+        '''
-    	'''
         Method for making a fd member of a structure.
         '''
         new_fd = Field(self, module.get_type_name('INT32'), fd_name, True, False, False, None, True)
